python/artm/regularizers.py

```diff
@@ -964,3 +965,146 @@ def class_ids(self, class_ids):
+            for second_index, value in iteritems(indices_and_values):
+                self._config.first_vertex_index.append(first_index)
+                self._config.second_vertex_index.append(second_index)
+                self._config.edge_weight.append(value)
+
+    def _update_from_config(self, config):
+        self._edge_weights = {}
+        for f_index, s_index, value in zip(config.first_vertex_index, config.second_vertex_index, config.edge_weight):
+            if f_index not in self._edge_weights:
+                self._edge_weights[f_index] = {}
+            self._edge_weights[f_index][s_index] = value
+        if self._edge_weights == {}:
+            self._edge_weights = None
+
+    def __init__(self, name=None, tau=1.0, gamma=None, class_id=None, symmetric_edge_weights=False,
+                 topic_names=None, vertex_names=None, vertex_weights=None, edge_weights=None, config=None):
+        """
+        :param str name: the identifier of regularizer, will be auto-generated if not specified
+        :param float tau: the coefficient of regularization for this regularizer
+        :param float gamma: the coefficient of relative regularization for this regularizer
+        :param str class_id: name of class_id of special tokens-vertices
+        :param topic_names: list of names or single name of topic to regularize,\
+                            will regularize all topics if empty or None
+        :type topic_names: list of str or single str or None
+        :param edge_weights: information about edge weights of NetPLSA model, required.
+        :type edge_weights: dict, key - first token, value - dict with second tokens and float values
+        :param bool symmetric_edge_weights: use symmetric edge weights or not
+        :param list vertex_names: list of tokens-vertices of class_id modality, required.
+        :param list vertex_weights: list of weights of vertices, should has equal length with\
+                                    vertex_name, 1.0 values for all vertices will be used by default
+        :param config: the low-level config of this regularizer
+        :type config: protobuf object
+        """
+        BaseRegularizerPhi.__init__(self,
+                                    name=name,
+                                    tau=tau,
+                                    gamma=gamma,
+                                    config=config,
+                                    topic_names=topic_names,
+                                    class_ids=None,
+                                    dictionary=None)
+
+        self._class_id = None
+        if class_id is not None:
+            self._config.class_id = class_id
+            self._class_id = class_id
+        elif config is not None and config.HasField('class_id'):
+            self._class_id = config.class_id
+
+        self._symmetric_edge_weights = False
+        if symmetric_edge_weights is not None:
```

Comment:
I've recently noticed that in arguments list you set default value for `symmetric_edge_weights`  as `False`. I think it's not OK, as in that case we never execute branch with `config`.